### PR TITLE
Logger implemented

### PR DESCRIPTION
**Pull request by Priyankagaikwad1995**

### CODE REVIEW

**Comment by jaydeepsankpal** [comment]
Please resolve conflicts.

File: src/main/java/com/programs/projects/vehicle/controller/EmployeeController.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import org.springframework.util.MimeTypeUtils;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+@RestController
+public class EmployeeController {
+    private static final Logger logger = LoggerFactory.getLogger(EmployeeController.class);
+
+    @Autowired
+    private EmployeeRepository employeeRepository;
+
```
Comment (jaydeepsankpal) [inline src/main/java/com/programs/projects/vehicle/controller/EmployeeController.java:17]: remove unwanted lines

File: src/main/java/com/programs/projects/vehicle/controller/EmployeeController.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    }
+    @PostMapping(value = ADD_EMPLOYEE, produces = {MimeTypeUtils.APPLICATION_JSON_VALUE, MimeTypeUtils.APPLICATION_XML_VALUE})
+    public void addVehicle(@RequestBody Employee employee) {
+        employeeService.addEmployee(employee);
+    }
+
+    @PutMapping (value = UPDATE_EMPLOYEE, produces = {MimeTypeUtils.APPLICATION_JSON_VALUE, MimeTypeUtils.APPLICATION_XML_VALUE})
+    public void updateVehicle(@RequestBody Employee employee) {
+        employeeService.updateEmployee(employee);
+
```
Comment (jaydeepsankpal) [inline src/main/java/com/programs/projects/vehicle/controller/EmployeeController.java:47]: remove unwanted lines

File: src/main/java/com/programs/projects/vehicle/controller/EmployeeController.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    }
+
+    @DeleteMapping(value = DELETE_EMPLOYEE, produces = {MimeTypeUtils.APPLICATION_JSON_VALUE, MimeTypeUtils.APPLICATION_XML_VALUE})
+    public void deleteVehicle(@RequestBody Employee employee) {
+        employeeService.deleteEmployee(employee);
+
```
Comment (jaydeepsankpal) [inline src/main/java/com/programs/projects/vehicle/controller/EmployeeController.java:53]: remove unwanted lines

File: src/main/java/com/programs/projects/vehicle/model/Employee.java [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+        this.empAddress = empAddress;
+        this.empDesignation = empDesignation;
+        this.empType = empType;
+        this.remark = remark;
+    }
+
+
+    public Employee() {
+    }
+
```
Comment (jaydeepsankpal) [inline src/main/java/com/programs/projects/vehicle/model/Employee.java:83]: Implement toString method

File: src/main/java/com/programs/projects/vehicle/repo/EmployeeRepository.java [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+import com.programs.projects.vehicle.model.Vehicle;
+import org.springframework.stereotype.Repository;
+
+import java.util.*;
+
+import static java.util.Optional.ofNullable;
+
+@Repository
+public class EmployeeRepository {
+
```
Comment (jaydeepsankpal) [inline src/main/java/com/programs/projects/vehicle/repo/EmployeeRepository.java:14]: add logs in this repository

File: src/main/java/com/programs/projects/vehicle/service/EmployeeService.java [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+    @Autowired
+    private EmployeeRepository employeeRepository;
+
+    public List<Employee> getAllEmployee() {
+        return employeeRepository.getAllEmployee();
+    }
+
+    public Employee getEmployeeByID(String id) {
+        return null;
```
Comment (jaydeepsankpal) [inline src/main/java/com/programs/projects/vehicle/service/EmployeeService.java:24]: Implement this method and add logs

File: src/main/resources/logback.xml [hunk 7]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+
+    <property name="HOME_LOG" value="logs/app.log"/>
+
+
```
Comment (jaydeepsankpal) [inline src/main/resources/logback.xml:6]: remove unwanted lines

File: src/main/java/com/programs/projects/vehicle/service/EmployeeService.java (hunk 6 above)
Comment (Priyankagaikwad1995) [reply src/main/java/com/programs/projects/vehicle/service/EmployeeService.java:24]: Implemented

File: src/main/java/com/programs/projects/vehicle/controller/EmployeeController.java (hunk 1 above)
Comment (Priyankagaikwad1995) [reply src/main/java/com/programs/projects/vehicle/controller/EmployeeController.java:17]: removed

File: src/main/java/com/programs/projects/vehicle/controller/EmployeeController.java (hunk 2 above)
Comment (Priyankagaikwad1995) [reply src/main/java/com/programs/projects/vehicle/controller/EmployeeController.java:47]: removed

File: src/main/java/com/programs/projects/vehicle/controller/EmployeeController.java (hunk 3 above)
Comment (Priyankagaikwad1995) [reply src/main/java/com/programs/projects/vehicle/controller/EmployeeController.java:53]: removed

**Pull request merged**